third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -627,7 +627,7 @@ func resourceSqlDatabaseInstanceCreate(d *schema.ResourceData, meta interface{})
 		for _, u := range users.Items {
 			if u.Name == "root" && u.Host == "%" {
 				err = retry(func() error {
-					op, err = config.clientSqlAdmin.Users.Delete(project, instance.Name, u.Host, u.Name).Do()
+					op, err = config.clientSqlAdmin.Users.Delete(project, instance.Name).Do()
```

Comment:
i'd update the description to explain why we're doing this here, or I'd move it to the vendor